lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -551,12 +552,33 @@ def get(self, request, course_key):
                     q_objects.append(Q(user__in=[]))
             if request.GET.get('enrollment_mode'):
                 q_objects.append(Q(mode=request.GET.get('enrollment_mode')))
+            if request.GET.get('assignment') and (
+                    request.GET.get('assignment_grade_max')
+                    or request.GET.get('assignment_grade_min')):
+                subqueryset = PersistentSubsectionGrade.objects.annotate(
+                    effective_grade=Case(
+                        When(override__isnull=False, then='override__earned_all_override'),
+                        default='earned_all'))
+                grade_conditions = {}
+                if request.GET.get('assignment_grade_max'):
+                    grade_conditions['effective_grade__lte'] = request.GET.get('assignment_grade_max')
+                if request.GET.get('assignment_grade_min'):
+                    grade_conditions['effective_grade__gte'] = request.GET.get('assignment_grade_min')
```

Comment:
I wonder if it'd be clearer to do `grade_conditions['effective_grade__range'] = (request.GET.get('assignment_grade_min', 0), request.GET('assignment_grade_max', 100))`

Reply:
That would work if the semantics for these inputs were percentages. As written, they're absolute scores. I could use an `F()` to make the default value be the max possible, but I think that'd take it into the realm of not being as readable as what we have

Reply:
The default could be `sys.maxint`... 😆